experimental/traffic-portal/src/app/guards/authenticated-guard.service.ts

```diff
@@ -12,15 +12,14 @@
 * limitations under the License.
 */
 import { Injectable } from "@angular/core";
-import type { CanActivate, CanLoad } from "@angular/router";
 
 import { CurrentUserService } from "src/app/shared/current-user/current-user.service";
 
 /**
  * AuthenticationGuard ensures that the user is logged in.
  */
 @Injectable()
-export class AuthenticatedGuard implements CanActivate, CanLoad {
+export class AuthenticatedGuard  {
```

Comment:
did the upgrade make this change as a migration?

Reply:
Yes, these don't exist anymore.